app/Http/Controllers/Api/V1/UserController.php

```diff
@@ -324,7 +326,7 @@ public function destroy(User $user)
     public function listNotifications(Request $request)
     {
         return response([
-            'notifications' => auth()->user()->unreadNotifications,
+            'notifications' => NotificationResource::collection(auth()->user()->unreadNotifications),
```

Comment:
Now that the response is updated, you also need to update doc block.
Do it for both functions.